### ISSUE TITLE
Readme images not found

### ISSUE THREAD

**Issue by pickfire** [NerdyPepper/taizen#18]
The images in the readme.md can't be found.
One points to a service called 0x0 that seems to have issues, the other link points to nerdypepper.me which also seems to not hold the images anymore.

**Comment by oppiliappan** [comment]
I have plans to rework this entire project, some of the changes i have in mind are:

 - MSRV 1.40+
 - async requests
 - use an actual parser to parse and display mediawiki formatted text

Ill update the readme with fresh images once this is done, although, I am not sure if I will have the time to in the near future.

**Comment by Primajin** [comment]
I find it a bit of an irony that the images for a text based browser don't work 😜

**Comment by pickfire** [comment]
> I find it a bit of an irony that the images for a text based browser don't work stuck_out_tongue_winking_eye

That is unrelated to this issue, you could open another issue for it. And I don't think cursive supports that, maybe you could also open an upstream issue.

**Comment by Primajin** [comment]
I am sorry if my humorous pun could not be conveyed or was not perceived as such. I'll open another issue then.

**Comment by pickfire** [comment]
@Primajin Oh wait, no need to open a new issue. The new issue you opened is a duplicated of this issue.

Sorry, I was confused. I thought you were talking about the support to display images in taizen which should be a new issue.

**Comment by Primajin** [comment]
Alrighty, no worries - would you mind adding a descrption to the first post something like:

> The images in the readme.md can't be found.
> One points to a service called 0x0 that seems to have issues, the other link points to nerdypepper.me which also seems to not hold the images anymore.

**Comment by ChristopherBilg** [comment]
Fixed by #25 :+1:
